src/api/app/components/notification_notifiable_link_component.rb

```diff
@@ -123,7 +124,7 @@ def notifiable_link_path
     when 'Event::ReportForProject', 'Event::ReportForPackage'
       @notification.event_type.constantize.notification_link_path(@notification)
     when 'Event::ReportForUser'
-      Rails.application.routes.url_helpers.user_path(@notification.event_payload['user_login'], notification_id: @notification.id)
+      Rails.application.routes.url_helpers.user_path(@notification.event_payload['user_login'], notification_id: @notification.id) if !@notification.event_user.is_deleted? || @current_user.is_admin?
```

Comment:
What happens if the `@notification.event_user` is _really_ deleted?

Reply:
NVM, I saw some `delete!` methods being used on user instances, looks like those are custom methods that soft delete users in the end...

Reply:
Ah, you probably mean _destroy_ AKA not in the database? That never happens https://github.com/openSUSE/open-build-service/pull/15669#discussion_r1499047824